scapy/arch/linux.py

```diff
@@ -438,34 +438,26 @@ def close(self):
             for i in self.iff:
                 set_promisc(self.ins, i, 0)
         SuperSocket.close(self)
-    def recv(self, x=MTU):
+    def recv_async(self, x=MTU):
```

Comment:
Not sure if this name is really appropriate... The goal of this function is to receive without dissecting. Any ideas?